elements/rh-progress-stepper/rh-progress-step.css

```
@@ -0,0 +1,148 @@
+    --_step-line-color: var(--rh-color-border-strong) !important;
+  }
+}
+
+#container {
+  display: grid;
+  grid-template-columns: var(--_step-template-columns);
+  grid-template-rows: var(--_step-template-rows, 24px) auto;
+  grid-template-areas: var(--_step-template-areas);
+  gap: var(--_step-row-gap, 16px) var(--_step-column-gap, 16px);
+  justify-self: var(--_step-justify-self, start);
+  align-items: var(--_step-align-items);
+  padding-block-end: var(--_step-compact-vertical-padding);
+  padding-inline-end: var(--_step-compact-horizontal-padding);
+  width: 100%;
+  height: 100%;
+  background: var(--_step-line-style);
+  color: var(--rh-color-text-secondary);
+
+  /* Body text/md */
+  font-family: var(--rh-font-family-body-text, RedHatText, 'Red Hat Text', Helvetica, Arial, sans-serif);
+  font-size: var(--rh-font-size-body-text-md, 1rem);
+  font-style: normal;
+  line-height: 1.5; /* 150% */
+
+  @container (min-width: 992px) {
+    padding-inline-end: var(--_step-compact-horizontal-padding, var(--rh-length-5xl, 80px));
+  }
+}
+
+/* Icon styling */
+:is(rh-icon, .no-icon) {
+  grid-area: icon;
+  place-self: center var(--_step-icon-justify-self);
+  height: var(--rh-size-icon-02, 24px);
+  width: var(--rh-size-icon-02, 24px);
+  border: var(--rh-border-width-md) solid var(--rh-color-text-primary);
+  border-radius: 50%;
+  color: var(--_step-color);
+  background-color: var(--rh-color-surface);
+}
+
+#label {
+  display: grid;
+  grid-area: label;
+  align-self: center;
+  color: var(--rh-color-text-primary);
+  font-weight: var(--_current-step-weight, var(--rh-font-weight-body-text-regular, 400));
+
+  &:is(a[href]) {
+    text-decoration:
+      underline
+      dotted
+      var(--rh-color-text-secondary)
+      1px;
+    text-decoration-skip-ink: none;
+    text-underline-offset: 5px;
+  }
+}
+
+#description {
+  grid-area: description;
+  display: grid;
+  margin: 0;
+  grid-column: var(--_step-description-grid-column, 1);
+  font-weight: var(--rh-font-weight-body-text-regular, 400);
+  font-size: var(--rh-font-size-body-text-xs, 0.75rem);
+}
+
+/* Compact view - hide text content */
+.compact {
+  & :is(#label, #description) {
+    /* Visually Hiding */
+    block-size: 1px;
+    border: 0;
+    clip: rect(0, 0, 0, 0);
+    inline-size: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    white-space: nowrap;
+
+    /* End Visually Showing */
+  }
+}
+
+/* Inactive state styling */
+:host([state='inactive']) {
+  :is(rh-icon, .no-icon) {
+    border-color: var(--rh-color-border-subtle);
+  }
+}
+
+/* Active state styling */
+:host([state='active']) {
+  --_step-color: var(--rh-color-status-note);
+}
+
+/* Complete state styling */
+:host([state='complete']) {
+  --_step-color: var(--rh-color-status-success);
+}
+
+/* Warning state styling */
+:host([state='warn']) {
+  --_step-color: light-dark(var(--rh-color-yellow-50), var(--rh-color-yellow-30));
```

```suggestion
  --_step-color:
    light-dark(
        var(--rh-color-yellow-50, #b98412),
        var(--rh-color-yellow-30, #ffcc17)
      );
```

Good to have fallbacks.